### PR TITLE
Minimal open-source documentation

### PR DESCRIPTION
**Pull request by coot**
- Use Apache-2.0 license
- Added a basic README.md file
- Include a minimal CONTRIBUTING.md file

### CODE REVIEW

**Comment by jbgi** [comment]
FYI, there is a CONTRIBUTING.md in this PR I'm working on: https://github.com/input-output-hk/cardano-prelude/pull/176/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055
(I'll rebase my PR on top of this one and merge both)

**Comment by coot** [comment]
ok, let's merge your PR first and then this one.  I'll merge the `CONTRIBUTING.md` files.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -128,3 +129,23 @@ You may also check what is the last `index-state` available within the current h
...
 ```
+
+## Roles and Responsibilities
+
+The `cardano-prelude` package is collectively mainntained by the `cardano-core`
+team.  If you open a PR expect a review from one of the core contributors.
+
+## CI
+
+`cardano-prelude` is build with GitHub Actions (using `cabal`) and cicero
```
Comment (newhoggy) [inline CONTRIBUTING.md:140]: s/build/built

**Review by newhoggy — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (coot) [reply CONTRIBUTING.md:140]: Actually this part should be removed, there already is a section about `CI`.

**Pull request merged**